### ISSUE TITLE
Issue with a steam api interface window not appearing visible in unity

### ISSUE THREAD

**Issue by cephalo3** [rlabrecque/Steamworks.NET#560]
I believe this might be an issue for steamworks.net. I am trying to implement Game Notifications for asynchronous multiplayer, and I'm stuck on one thing.

When I call this web api function [RequestNotifications](https://partner.steamgames.com/doc/webapi/IGameNotificationsService#RequestNotifications) from my server, there is a dialog box that is supposed to pop up and tell steam whether it's ok to receive game notifications. Like a callback, but one that steam calls instead of my client.

When I call this correctly from my server, my game becomes unresponsive, as if it is waiting for me to respond to something. None of my other buttons on screen work. However, when I hit shift+tab, things go back to normal and I can use the app.

I believe my app is being called by steam, because of the freeze, but unity is preventing the screen from being visible or interactable. Pressing shift+tab seems to close the window with a negative response.

The shift+tab screen works just fine, and achievement notifications works just fine, but I wonder if my failure to show the desired screen is related somehow. Maybe I'm not setting up my client to handle this particular event.

**Comment by cephalo3** [comment]
Here is my letter to Steam Support, it contains more information about my experiments:

Hi I am trying to figure out how to use the IGameNotificationsService/RequestNotifications web API. I have things working on the backend by properly getting a steam id from the web api authentication ticket and using that to call the web function.

My app is responding, but not in a way that I expected. It’s clear that something is hooking my interface on the client, but it’s not visible. Let me describe the behavior I have observed with my testing. 

When I initiate the process of calling RequestNotifications, my UI screen in my game is no longer usable, as if something is blocking it. If I press shift+tab as if closing the steam overlay, I can then use my UI as normal, so there is some kind of invisible process hooking into my game.

I thought maybe this was part of the steam overlay, so I activated the overlay before calling the server, and the normal steam overlay works just fine, but when I activate RequestNotifications, the steam overlay goes invisible and again the app is unresponsive until I press shift+tab.

I thought that maybe my game graphics were overwriting this special RequestNotifications overlay, but when I stop everything from drawing, my steam overlay comes up again but I still do not see any user interface from this alternate overlay that turns off or replaces the normal steam overlay.

Clearly, the client has certain responsibilities in responding to IGameNotificationsService/RequestNotifications that are not in the documentation. Why is my steam overlay and other friend notifications working just fine, while this other overlay is not? How am I supposed to use this web api function? What am I not doing that needs to be done? Is there a callback I need to handle?
